icpc-wf/ansible/group_vars/all/secret.yml.example

```diff
@@ -17,3 +17,32 @@ ADMIN_PASSWORD: some-admin-password
 # Set this to enable a password on the 'domjudge' shell accounts
 # created on the domserver and judgehosts.
 #DJ_SHELL_USER_PW: some-hashed-password
+
+# Users to create when setting up the CDS
+CDS_USERS:
+    - username: admin
+      password: adm1n
+    - username: presAdmin
+      password: padm1n
+    - username: blue
+      password: blu3
+    - username: balloon
+      password: balloonPr1nter
+    - username: public
+      password: publ1c
+    - username: presentation
+      password: presentat1on
+    - username: myicpc
+      password: my1cpc
+    - username: live
+      password: l1ve
+    - username: team1
+      password: t3am
+
+# Contest(s) to configure in the CDS
+CDS_CONTESTS:
+  - path: nwerc18 # Path in the contest directory
+    ccs:
+      url: https://www.domjudge.org/demoweb/api/contests/nwerc18
```

Comment:
Should this reuse `DOMSERVER_URL`?

Reply:
I didn’t do this on purpose: if we have a shadow system that we also talk to, it is/can be a different url

Reply:
We could make it more fancy that you can either provide the url or a DJ ID, but I’m not sure if that’s worth it?

Reply:
I did make it 'more fancy', indeed this is better.